examples/nginx-access-logs-streaming/README.md

```diff
@@ -0,0 +1,217 @@
+# Analyzing Nginx Access Logs using Timeplus Proton 
+In a [recent article](https://www.timeplus.com/post/log-stream-analysis), we walked through three different ways of using Timeplus Proton to ingest and analyse a log file in real-time.
+
+I’ll explore a similar theme in the first part of this article: ad-hoc analysis of web traffic, in real-time, using Timeplus Proton. I will use a Node.js blog that is behind an Nginx web server for the analysis. 
```

Comment:
Maybe just me, but the context is a bit too long for first part, then 2nd part. Maybe something like "In this blog, I will talk about.... (then a bullet point) the first part will be .. (2nd bullet point) the second part .."

"a Node.js blog" maybe a blog web application built by Node.js" or something like that

Reply:
Fixed.